static/js/github-configuration.js

```diff
@@ -1,17 +1,28 @@
 /* globals $ */
 $('.install-link').click(function (event) {
   event.preventDefault();
+	event.currentTarget.setAttribute('disabled', true);
+	hideErrorMessage();
 
-  $.post(window.location.href, {
-    installationId: $(event.target).data('installation-id'),
-    _csrf: document.getElementById('_csrf').value,
-    clientKey: document.getElementById('clientKey').value
-  }, function (data) {
-    if (data.err) {
-      console.log(data.err);
-    }
-    window.close();
-  })
+	$.post(window.location.href, {
+		installationId: $(event.target).data('installation-id'),
+		_csrf: document.getElementById('_csrf').value,
+		clientKey: document.getElementById('clientKey').value
+	}, function (data) {
+		event.currentTarget.removeAttribute("disabled");
+		if (data.err) {
+			showErrorMessage(data.err);
+		}
+		window.close();
+	}).fail((err) => {
+		event.currentTarget.removeAttribute("disabled")
```

Comment:
/nit
Either more defensive programming like `if(err.resposneJSON)` or use a `try/catch` in case we have error showing error?